docs/admin/default-policies.md

```diff
@@ -1,25 +1,33 @@
 # Default policies
 
 A set of policies can also be set up to be created upon the creation of
-a Tenant through a file.
+a Tenant through a file, which represents a series of default policies for the
+root servicepath `/`. The file is turtle (ttl) format, representing an RDF
+graph.
+
 See this [example](https://github.com/orchestracities/anubis/blob/master/config/opa-service/default_policies.yml).
 
-```yaml
-# TODO: This will eventually be a different more fitting wac format
-"acl:Authorization":
-  - "acl:accessTo":
-      value: "default"
-      type: "*"
-    "acl:agentClass":
-      - "acl:AuthenticatedAgent"
-    "acl:mode":
-      - "acl:Read"
-  - "acl:accessTo":
-      value: "default"
-      type: "*"
-    "acl:agentClass":
-      - "acl:agentClass:Admin"
-    "acl:mode":
+```ttl
+@prefix acl: <http://www.w3.org/ns/auth/acl#> .
+@prefix tenant: <https://tenant.url/> .
+
+tenant:policy1 a acl:Authorization ;
+    acl:agentClass <acl:agentClass:Admin> ;
+    acl:default </> ;
+    acl:accessToClass <entity> ;
```

Comment:
From the `default_wac_config.yml` file, you have something like this:

```yml
Tenant1:
      prefix: tenant1
      baseUrl: https://tenant1.url/
      resourceTypeUrls:
        entity:
          url: https://tenant1.orion.url/v2/entities/
          type_url: https://tenant1.url/entity
        entityType:
          url: https://tenant1.orion.url/v2/types/
          type_url: https://tenant1.url/entity_type
        policy:
          url: https://tenant1.anubis.url/v1/policies/
          type_url: https://tenant1.url/policy
```

Since the definition of the type of the resource and the resource itself are obviously on different URIs that should be configured separately.

Reply:
what i meant is rather simple. suppose that the tenant is the same (which is not in the current example, and then we should make things consistent). 

in the moment i have a configuration that says that 
```
resourceTypeUrls:
   entity:
      url: https://tenant1.orion.url/v2/entities/
      type_url: https://tenant1.url/entity
```
 and i am parsing a WAC rule that is:
 ```
@prefix acl: <http://www.w3.org/ns/auth/acl#> .
@prefix tenant1: <https://tenant1.url/> .
tenant:policy1 a acl:Authorization ;
    acl:agentClass <acl:agentClass:Admin> ;
    acl:accessTo <https://tenant1.orion.url/v2/entities/pippo> ;
    acl:accessToClass <entity> ;
 ```

this should be translated "internally" into:
```
acl:accessTo <pippo> ;
```

or otherwise when you serialize back, you will have:

```
@prefix acl: <http://www.w3.org/ns/auth/acl#> .
@prefix tenant1: <https://tenant1.url/> .
tenant:policy1 a acl:Authorization ;
    acl:agentClass <acl:agentClass:Admin> ;
    acl:accessTo <https://tenant1.orion.url/v2/entities/https://tenant1.orion.url/v2/entities/pippo> ;
    acl:accessToClass <entity> ;
```

or am I wrong?